testsuite/config/tools.py

```diff
@@ -48,6 +52,23 @@ def _fetcher(settings, _):
     return _fetcher
 
 
+def fetch_service_ip(name, port, force_http=False):
+    """Fetched load balanced ip for LoadBalancer service"""
+
+    def _fetcher(settings, _):
+        try:
+            openshift = settings["tools"]
+            with openshift.context:
+                ip = selector(f"service/{name}").object(cls=Service).external_ip
+                return f"http://{ip}:{port}" if force_http else f"https://{ip}:{port}"
+        # pylint: disable=broad-except
+        except Exception:
+            logger.warning("Unable to fetch route %s from tools", name)
+            return None
```

Comment:
I think there is no reason to have broad exception here. It should basically be able to produce only 2 errors - Openshift one and AttributeError, am I wrong?
```suggestion
        try:
            openshift = settings["tools"]
            with openshift.context:
                ip = selector(f"service/{name}").object(cls=Service).external_ip
                return f"http://{ip}:{port}" if force_http else f"https://{ip}:{port}"
        # pylint: disable=broad-except
        except (OpenshiftPythonError, AttributeError):
            logger.warning("Unable to fetch route %s from tools", name)
            return None
```

Reply:
Theoretically yes, but it is there to be safe because raising exception here would mean failing whole testsuite

Reply:
It's fine. I think testsuite should raise errors on unexpected code paths